appside/data/constants.js

```diff
@@ -33,25 +33,36 @@ var SVG_URL = 'http://www.w3.org/2000/svg';
 
 // Intro frame strings
 var INTRO_TITLE = {};
-INTRO_TITLE[SECTION_PROMPTING] = 'Prompting Events fwd (variant 1af)';
-INTRO_TITLE[SECTION_INTERP] = 'Interperetations of Events fwd (variant 1bf)';
-INTRO_TITLE[SECTION_BIO] = 'Biological Changes and Experiences fwd (variant 1cf)';
-INTRO_TITLE[SECTION_ACT] = 'Expressions and Actions fwd (variant 1df)';
-INTRO_TITLE[SECTION_AFTER] = 'Aftereffects fwd (variant 1ef)';
+INTRO_TITLE[SECTION_PROMPTING] = 'Emotional Clarity App';
+INTRO_TITLE[SECTION_INTERP] = 'Emotional Clarity App';
+INTRO_TITLE[SECTION_BIO] = 'Emotional Clarity App';
+INTRO_TITLE[SECTION_ACT] = 'Emotional Clarity App';
+INTRO_TITLE[SECTION_AFTER] = 'Emotional Clarity App';
```

Comment:
good suggestion. I added a variable for the string `'Emotional Clarity App'`, so as to avoid repeating the same string. Worthwhile to leave the option for setting different titles in the future, so I keeping `INTRO_TITLE` as an object rather than a single primitive.